### DISCUSSION TITLE
Question regarding IPI

### DISCUSSION THREAD

**Discussion by stickynoteinsanity** [amzn/selling-partner-api-models#3247] [General]
Is there an API that allows access to the data related to Inventory Performance Index on Seller Central?

**Comment by stefnats** [comment]
Hey there,

i think i found something:
https://github.com/amzn/selling-partner-api-docs/blob/main/references/reports-api/reporttype-values.md#performance-reports

I think via Reports API you could request the `GET_V1_SELLER_PERFORMANCE_REPORT`  Report Type:

> XML Customer Metrics Report
> XML report that contains select, individual performance metrics data from the Seller Central dashboard. For all sellers.

I hope this is what you're looking for.
